library/src/network.rs

```diff
@@ -162,15 +248,39 @@ pub fn send_patch_check_request(
         platform: current_platform().to_string(),
         arch: current_arch().to_string(),
     };
-    debug!("Sending patch check request: {:?}", request);
```

Comment:
why did we remove the debug logs?

Reply:
My mistake, thought I added those in my testing :)